src/store/modules/notifications/notifications.spec.ts

```diff
@@ -1,23 +1,38 @@
-import { createStore } from 'vuex'
+// import { createStore } from 'vuex'
```

Comment:
if it's really not needed this should be removed

Reply:
Cleaned this up.